client/src/components/Sidebar.js

```diff
@@ -1,53 +1,54 @@
-import React, { useEffect, useState } from 'react';
-import { Outlet, useNavigate, useParams } from 'react-router-dom';
-import SideBarPopupAdmin from './SideBarPopupAdmin';
-import AnnouncementIcon from '../shared/AnnouncementIcon';
-import UserApi from '../api/UserApi';
-import ROLES from './Roles';
-import Cookies from 'js-cookie';
-import CollegeAccordion from './CollegeAccordion';
-import SideBarPopupUser from './SidebarPopupUser';
+import React, { useEffect, useState } from "react";
+import { Outlet, useNavigate, useParams } from "react-router-dom";
+import SideBarPopupAdmin from "./SideBarPopupAdmin";
+import AnnouncementIcon from "../shared/AnnouncementIcon";
+import UserApi from "../api/UserApi";
+import ROLES from "./Roles";
+import Cookies from "js-cookie";
+import CollegeAccordion from "./CollegeAccordion";
+import SideBarPopupUser from "./SidebarPopupUser";
 
 const Sidebar = () => {
   const id = useParams();
   const navigate = useNavigate();
   const [userData, setUserData] = useState([]);
-  const user = JSON.parse(Cookies.get('user') || '{}');
+  const user = JSON.parse(Cookies.get("user") || "{}");
   const [Show, setShowModal] = useState(false);
+
   useEffect(() => {
     UserApi.fetchUser().then((res) => {
       setUserData(res.data);
     });
-  }, []);
+  }, [id]);
 
   return (
     <div className="flex shrink-0 w-full h-screen">
       <div className="flex flex-col w-full overflow-y-auto min-w-[240px] max-w-[240px] justify-between bg-regal-blue text-white">
         <div className="mb-20">
           <div
-            onClick={() => navigate('/')}
-            className="flex justify-center px-2 my-5 cursor-pointer"
+            onClick={() => navigate("/")}
+            className="flex justify-center p-4 cursor-pointer sticky top-0 bg-regal-blue border-b-2"
```

Comment:
Can you add a padding or margin on this one sir? I think its too close

![image](https://user-images.githubusercontent.com/109492180/197674079-ea6ab4f9-584f-4e92-9714-3c58359e1b38.png)